Zilon.Core/Zilon.Core.Specs/Contexts/FeatureContextBase.cs

```diff
@@ -176,18 +129,19 @@ public Equipment CreateEquipment(string propSid)
 
         public void AddHumanActor(string personSid, OffsetCoords startCoords)
         {
+            var humanPlayer = ServiceProvider.GetRequiredService<IPlayer>();
+
+            var sector = humanPlayer.SectorNode.Sector;
+
             var playerState = ServiceProvider.GetRequiredService<ISectorUiState>();
-            var sectorManager = ServiceProvider.GetRequiredService<ISectorManager>();
-            var humanTaskSource = ServiceProvider.GetRequiredService<IHumanActorTaskSource>();
-            var actorManager = sectorManager.CurrentSector.ActorManager;
-            var humanPlayer = ServiceProvider.GetRequiredService<HumanPlayer>();
+            var humanTaskSource = ServiceProvider.GetRequiredService<IHumanActorTaskSource<ISectorTaskSourceContext>>();
```

Comment:
Codacy found an issue: [Remove the unused local variable 'humanTaskSource'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6105716)